### PR TITLE
Ethers vs web3

### LINKED ISSUE #7023: Web3js vs EthersJS
### Topic Suggestion
Web3js vs EthersJS

### Writing sample(s): 
https://dev.to/ghoulkingr/simple-ways-to-compile-and-deploy-solidity-contracts-3kpb

#### Proposed title of article
Web3 vs Ethers

Use keyword research to improve your article's SEO.

#### Key takeaways 
* The readers will be able to tell the strengths and weaknesses of the modules
*  They will be able to know  the scenario where one is most applicable is most applicable
* They'll understand why the both of them exists

#### Article quality
This article helps the readers distinguish between the Web3js and the EthersJS module. In this article the readers will be given a clear understanding of the concepts that the two modules entail and they'll be able to understand why the both of them even exists

#### References
N/A

### PR DESCRIPTION
**Pull request by GhoulKingR**
#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

#### Topic suggestion this closes
This closes #7023

### CODE REVIEW

**Comment by louisefindlay23** [comment]
Please remove the extra line breaks between the paragraphs of your article.

**Comment by louisefindlay23** [comment]
Plagiarism check complete

**Review by ghost — COMMENTED** [review]
Great article @TheGhoulRe , finish the errors so we can move forward.

**Comment by TheGhoulRe** [comment]
Good day @mohamedgh16, I have fixed the errors. Please let me know if there are any concerns

**Comment by mohamedgh16** [comment]
Great article @TheGhoulRe please consider the corrections to minimize revision next time.

**Comment by TheGhoulRe** [comment]
@mohamedgh16 thanks for the corrections. I will consider them next time 👍

File: content/articles/web3js-vs-ethersjs/index.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+author: Chigozie Oduah
+date: 2022-04-5T00:00:00-02:30
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /ethers-vs-web3/hero.jpg
+    alt: Ethers vs web3 Hero Image
+---
+The web3js library and the ethers library are the major libraries for utilizing the Ethereum network in JavaScript. The two have different ways they interact with smart contracts and make transactions on the network. This difference leads to one handling some type of tasks better than the other. In this article, you will learn their differences, their best use cases, and how they both interact with the network.
```
Comment (WanjaMIKE) [inline content/articles/web3js-vs-ethersjs/index.md:17]: The web3js `library` and the ethers `library` are the major `libraries` for utilizing the Ethereum network in JavaScript  - You used library thrice which results in a poor sentence.

File: content/articles/web3js-vs-ethersjs/index.md [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+### Prerequisites
+To get the full benefits of this article, you need to have the following.
+- You need to have a working knowledge of JavaScript.
```
Comment (WanjaMIKE) [inline content/articles/web3js-vs-ethersjs/index.md:20]: No need to include the highlighted parts below:
- `You need` to have a working knowledge of JavaScript.
- `You should` have a basic understanding of Web3, Smart Contracts, and Blockchain.

**Comment by WanjaMIKE** [comment]
Could you check out my suggestions @TheGhoulRe

**Comment by TheGhoulRe** [comment]
@WanjaMIKE I've resolved the suggestions

**Comment by hectorkambow** [comment]
We will be reviewing this (& publishing if it is ready) ASAP.

**Comment by hectorkambow** [comment]
👏  

Please go over any edits made by your peer reviewer (and myself) to improve future blog posts. 

A skill section will go a long way for potential future employers !

**Pull request merged**